### ISSUE TITLE
gws status display the current branch

### ISSUE THREAD

**Issue by thomas-lemaire** [StreakyCobra/gws#42]
the `gws status` shows all branches and their status (*clean*, *dirty*,...). That would be nice to have some visual clue to show which branch is currently checked out

**Comment by emlun** [comment]
Good idea, I agree. I'm afraid I can't promise anything about when I'll have time to implement it, though. You're welcome to contribute the feature if you feel like it.

**Comment by thomas-lemaire** [comment]
I could have a look, *bash* is generally not my cup of tee

**Comment by thomas-lemaire** [comment]
ok, I just did a *pull request*, I let you check
